### ISSUE TITLE
Request for "Next element of different type" gesture

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#18543]
### Is your feature request related to a problem? Please describe.

This [originally came in](https://fosstodon.org/deck/@ke7zum@glitchsoc.bg-presents.us/114897993885511305) as a user unable to reliably skip past long graphic alt text to the next element.

Pressing control+down arrow or p to jump to the next paragraph may work in some cases, but not all.

### Describe the solution you'd like

In discussion with @SaschaCowley, the suggestion came up that a more general "jump to next element of different type" could be useful in a wider range of situations.

We currently have comma which specifically jumps to the next element beyond the current container (list, table, etc), so perhaps the dot or full stop "." key could work in this case.  Or the gesture could be left unassigned.

### Describe alternatives you've considered

As noted, control+down arrow or p may work in some cases.  If the next element is known, it may be possible to jump directly to it, however in the original use case of a long graphic alt text, this may occur anywhere without a consistent next element.

**Comment by Emil-18** [comment]
@Qchristensen What about treating any element that has more than one line as a container? This would also make it easier to detect that you are in the same element when using up or down arrows to navigate. As it is now, when I encounter things such as a button that has more than one line, I think it can be dificult to know if it is the same button, because NVDA says button on each line

**Comment by cary-rowen** [comment]
Expanding the capabilities of comma to allow them to jump out more elements is better than adding other commands.

**Comment by SaschaCowley** [comment]
That was @Qchristensen's initial suggestion, however I think that extending the "move past the end of container" command to jump past the end of any element would be a very confusing user experience, and would make the gesture less useful.

1. As an NVDA user, I want to be able to jump past the end of the current container, so that I can more easily skip past irrelevant information.
   * True, jumping past the end of any element fits this user story, but:
   * It makes working with websites like the Python Documentation, where there are long lists of information, less efficient. Also:
   * What elements do we consider? Should a run of **bold face** count as an element that you can skip to the end of?

**Comment by SaschaCowley** [comment]
We think that this proposal would result in a confusing user experience. If desired, this feature should be implemented in an add-on.
